jinseok/model/userDao.js

```diff
@@ -19,8 +19,70 @@ const createUser = async (
       `,
       [email, profileImage, password, name, age, phoneNumber]
     );
-  } catch (err) {
-    console.log(err);
+  } catch (e) {
+    console.log(e);
+    const error = new Error('INVALID_DATA_INPUT');
+    error.statusCode = 500;
+    throw error;
+  }
+};
+
+const isExistedUser = async (email) => {
+  try {
+    const [result] = await dataSource.query(
+      `
+      SELECT EXISTS (
+        SELECT
+        id
+        FROM users 
+        WHERE email = ?
+    ) idExists
+    `,
+      [email]
+    );
+    return parseInt(result.idExists);
```

Comment:
```suggestion
    return !!parseInt(result.idExists);
```
double bang을 붙여서 true false 값을 보내줘야지 해당 함수를 쓰는 부분에서 또다른 별도의 가공이 필요하지 않게 됩니다.